src/scalars/iso-date/DateTime.ts

```diff
@@ -10,10 +10,15 @@
 import { GraphQLScalarType, Kind } from 'graphql';
 import type { GraphQLScalarTypeConfig } from 'graphql'; // eslint-disable-line
 import { validateJSDate, validateDateTime } from './validator.js';
-import { parseDateTime } from './formatter.js';
+import {
+  serializeDateTime,
+  serializeDateTimeString,
+  serializeTimestamp,
+  parseDateTime,
+} from './formatter.js';
 import { createGraphQLError } from '../../error.js';
 
-export const GraphQLDateTimeConfig: GraphQLScalarTypeConfig<Date, Date> = /*#__PURE__*/ {
+export const GraphQLDateTimeConfig: GraphQLScalarTypeConfig<Date, string> = /*#__PURE__*/ {
   name: 'DateTime',
   description:
     'A date-time string at UTC, such as 2007-12-03T10:15:30Z, ' +
```

Comment:
The sample date here should be changed to ISO string representation of ECMAScript timestamp.
Meaning, the miliseconds should be added in sample.

```suggestion
    'A date-time string at UTC, such as 2007-12-03T10:15:30.123Z, ' +
```

Reply:
pushed in https://github.com/Urigo/graphql-scalars/pull/1641/commits/796aae36dc8acf35c7bf46a62332e037f0e595b1